src/Sylius/Behat/Context/Api/PaypalContext.php

```diff
@@ -165,7 +165,7 @@ private function postPaymentRequest(array $payment): void
         $authorize = $gatewayConfig->getConfig()['use_authorize'] ?? false;
 
         $request->setContent([
-            'paymentId' => $payment['@id'],
+            'paymentId' => $payment['id'],
```

Comment:
Does it not work with an IRI anymore?

Reply:
@NoResponseMate apparently it is not working anymore, maybe the default conten-type as changed ?

Reply:
Maybe it will work if we bring back payment as an standalone resource, now it throws error that it cannot be subresource

Reply:
Probably a limitation of the IRI converter;
We can take a look at it at a later date 👋